common/app/redux/utils.js

```diff
@@ -10,12 +10,30 @@ export function filterComingSoonBetaChallenge(
 }
 
 export function filterComingSoonBetaFromEntities(
-  { challenge: challengeMap, ...rest },
+  { challenge: challengeMap, block: blocks, ...rest },
```

Comment:
nit: this should stick to the convention of calling these id => object objects `maps`. We use the plural of the noun to represent iterables or streams. So `blocks` should be `blockMap`.